src/main/java/seedu/address/model/project/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.model.project;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Project's deadline.
```

Comment:
Perhaps the naming of variables and documentation within `Deadline` could be less localized to `Project`? Since it is to be used by `Issue` as well. Improvement can be extended to other classes such as `Name`.

Reply:
I agree, but because it is unclear where is the best place for it now, its placed there